tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="$(SolutionDir)ToolingVersions.props" />
```

Comment:
csproj files should have indents of 2 spaces. This used to happen by default but now a couple PRs have come in in a few days where they've had 4. I'll look into what changed but let's keep 2 since that's what all the other files have.

Reply:
My theory is that I did modify this in notepad++ and I have indents of 4 spaces.  I'll get this changed to 2.